src/wp-content/themes/twentytwentythree/theme.json

```diff
@@ -718,7 +723,7 @@
 		{
 			"area": "uncategorized",
 			"name": "comments",
-			"title": "Comments"
+			"title": "Comments Template Part"
```

Comment:
Same goes for this string. I think this makes things more clear, but I don't know that it warrants breaking hard string freeze.